### PR TITLE
Add NavLink to the Layout component

### PR DESCRIPTION
**Pull request by mikeyxx**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### CODE REVIEW

**Comment by KarenJParker77** [comment]
Hi Mike and Sandra...Cassandra and I have checked your code and tested the links and it all looks great to us. We wouldn't change anything!

**Review by edieblu — APPROVED** [review]
Good job on creating your first PR! 🎉🕺

All looks good; please go ahead and merge!

**Comment by smguerra** [comment]
Merged Nav tag PR

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { Outlet } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
```
Comment (pmbanugo) [inline src/views/Layout.jsx:2]: this is a duplicate import

File: src/views/Layout.jsx [hunk 2]
```diff
@@ -21,15 +22,15 @@ export function Layout() {
 					<Outlet />
 				</main>
 				<nav className="Nav">
-					<a href="#" className="Nav-link">
+					<NavLink to="/" className="Nav-link">
```
Comment (pmbanugo) [inline src/views/Layout.jsx:25]: Nice work!

**Review by pmbanugo — COMMENTED** [review]
Great Work

**Pull request merged**